department-of-veterans-affairs/health-apis-parent
# labbot improvements

- Fixed config file to support cross-platform
- IdMeOauthRobot now supports toggling oauth types
  - {client-id}:{client-secret} as request body
  - {client-id}:{client-secret} as request headers

To test:
- build the snapshot branch of health-apis-parent 
- update the data-query-tests pom to use the local parent
- run OauthLoginTest

Noticeable changes:
Alternating oauth techniques, see below...

2019-07-31 11:36:07 INFO  g.v.a.h.s.selenium.IdMeOauthRobot - Exchanging authorization code for token using basic authorization in request body
2019-07-31 11:36:07 INFO  g.v.a.h.s.selenium.IdMeOauthRobot - Exchanging authorization code for token using basic authorization in request body
2019-07-31 11:36:07 INFO  g.v.a.h.s.selenium.IdMeOauthRobot - Exchanging authorization code for token using basic authorization header
2019-07-31 11:36:07 INFO  g.v.a.h.s.selenium.IdMeOauthRobot - Exchanging authorization code for token using basic authorization in request body
2019-07-31 11:36:07 INFO  g.v.a.h.s.selenium.IdMeOauthRobot - Exchanging authorization code for token using basic authorization header
